ocs_ci/ocs/platform_nodes.py

```diff
@@ -1736,8 +1736,8 @@ def stop_nodes(self, nodes, force=True):
                 nodes, timeout=900, wait=True, force=force
             )
         else:
-            raise NotImplementedError(
-                "This is not libvirt environment. Stop nodes not implemented"
+            self.powernodes.stop_powernodes_machines_powervs(
+                nodes, timeout=900, wait=True, force=force
```

Comment:
`use_method=kube-service` is better option than `force`

Reply:
Vasu, see my other comments above.

Reply:
@gitsridhar I see there are 2 force options, I meant changing of the the force of stop_power and start_power functions to use `use_method` to distinguish between service and real power restarts

Reply:
@vasukulkarni I have simplified the service, There are no more two different 'force' parameters. There is only one now when the service object is initialized, the force parameter can be initialized.  Hope this clears the confusion over the force parameter.